app/models/miq_report.rb

```diff
@@ -99,7 +99,7 @@ def col_format_with_defaults
 
   def conditions=(exp)
     exp = MiqExpression.new(exp) if exp && !exp.kind_of?(MiqExpression)
-    super
+    _write_attribute("conditions", exp)
```

Comment:
`super` is the expected way you are supposed to override columns.  I'm surprised we need this change or that it works?